.gitignore

```diff
@@ -2,6 +2,7 @@
 /app/config/parameters.yml
 /build/
 /phpunit.xml
+/bin/*
```

Comment:
We have some important files in bin directory such as console file, maybe is better to be specific. What files do you want ignore?